examples/df.go

```diff
@@ -1,10 +1,11 @@
-package examples
+package main
 
 import (
 	"fmt"
 	"os"
+	"strings"
 
-	"github.com/cloudfoundry/gosigar"
+	sigar "github.com/cloudfoundry/gosigar"
```

Comment:
Why are you aliasing this import? (Or whatever this operation is actually called in the golang world.)

The code works fine without it (tested on Go 1.19.5), and you _don't_ do it to `examples/uptime.go`. I'd rather not see this this change in this file, or in `examples/free.go` or `examples/ps.go`.

Reply:
This comment has been made obsolete by commit 1f172c217176871a86e495d319bcf54758bea6ab

Reply:
This is done by [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports). I've now done this on all files to make it consistent. I can also remove it if you want.